server_plugin/server.py

```diff
@@ -343,8 +352,13 @@ def _create_connections_list(vca_client):
 
     for port in ports:
         port_properties = port.node.properties['port']
+        network_name = port_properties['network']
```

Comment:
I think this change is redundant. The network name in port properties is mandatory.